fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1002,7 +1010,7 @@ public int getDegreeOfParallelism() {
             if (pipelineDop > 0) {
                 return pipelineDop;
             }
-            return BackendCoreStat.getDefaultDOP();
+            return Math.min(maxPipelineDop, BackendCoreStat.getDefaultDOP());
```

Comment:
If `maxPipelineDop` <= 0, should we use `BackendCoreStat.getDefaultDOP()` or `1`?

Reply:
> If `maxPipelineDop` <= 0, should we use `BackendCoreStat.getDefaultDOP()` or `1`?

I think we should use `BackendCoreStat.getDefaultDOP()`. I will fix it soon